packages/dev/docs/src/Layout.js

```diff
@@ -68,7 +68,14 @@ const mdxComponents = {
   p: ({children, ...props}) => <p className={typographyStyles['spectrum-Body3']} {...props}>{children}</p>,
   ul: ({children, ...props}) => <ul {...props} className={typographyStyles['spectrum-Body3']}>{children}</ul>,
   ol: ({children, ...props}) => <ol {...props} className={typographyStyles['spectrum-Body3']}>{children}</ol>,
-  code: ({children, ...props}) => <code {...props} className={typographyStyles['spectrum-Code4']}>{children}</code>,
+  code: ({children, ...props}) => (
+    <>
+      <button className={clsx(buttonStyles['spectrum-ActionButton', 'spectrum-ActionButton--sizeM', 'spectrum-ActionButton--quiet'], 'codeSandboxButton')} style={{float: 'right'}}>
+        <span className={buttonStyles['spectrum-ActionButton-label']}>Open Sandbox</span>
+      </button>
```

Comment:
IMO this button could use a more descriptive `aria-label` (maybe "Open current example in a code sandbox"?), currently just states "Open Sandbox button" which might be confusing w/o context

Reply:
Fixed, thanks!